### ISSUE TITLE
Consider adding back newlines in toZpl()

### ISSUE THREAD

**Issue by NiklasBr** [robgridley/zebra#10]
In commit 42e48e2f6ca354315c4411765778c5b1ae1ef402 you removed the newlines from output, I don't know why, but guess it has to do with output size. But can I ask you to consider adding the line breaks back, or at least the possibility to use line breaks. They have been invaluable to us when diff:ing 
output to find subtle changes.

For example, maybe make it something like this?
```php
public function toZpl($implode = '')
{
    return implode($implode, array_merge(['^XA'], $this->zpl, ['^XZ']));
}
```

**Comment by robgridley** [comment]
It was to shave a few bytes off of the output size on complex labels. I didn't consider that anyone would be diff-ing the output from this library. I have added a newlines argument to the `Builder::toZpl()` method and released version [2.0.1](https://github.com/robgridley/zebra/releases/tag/2.0.1).

**Comment by NiklasBr** [comment]
Super, thanks!
